pkg/webhook/v1beta1/pod/utils.go

```diff
@@ -241,12 +254,15 @@ func mutateVolume(pod *v1.Pod, mountPath, sidecarContainerName, primaryContainer
 	}
 
 	for _, i := range indexList {
-		c := &pod.Spec.Containers[i]
-		if c.VolumeMounts == nil {
-			c.VolumeMounts = make([]v1.VolumeMount, 0)
+		if err := addContainerVolumeMount(&pod.Spec.Containers[i], &vm); err != nil {
+			return err
```

Comment:
I think `addContainerVolumeMount()` always returns `nil`. Do we need to check errors?

(Same with `pkg/webhook/v1beta1/pod/inject_webhook.go`)